src/DotLiquid/Tags/AddFilters.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Text.RegularExpressions;
+using DotLiquid.Exceptions;
+using DotLiquid.Util;
+
+namespace DotLiquid.Tags
+{
+    /// <summary>
+    /// Adds filters in a whitelisted class to the current context.
+    ///
+    /// {%- addfilters 'ShopifyFilters' -%}
+    /// </summary>
+    public class AddFilters : Tag
+    {
+        private static readonly Regex Syntax = R.B(R.Q(@"\s*([""'].+[""'])\s*"));
+
+        private static readonly IDictionary<string, Type> FILTER_WHITELIST = new Dictionary<string, Type>(StringComparer.OrdinalIgnoreCase);
+        static AddFilters()
+        {
+            WhitelistFilter(typeof(ShopifyFilters));
+        }
+
+        /// <summary>
+        /// Add the provided class to the whitelist of classes that can be added by a Liquid Designer
+        /// </summary>
+        /// <param name="filterClassType">A class containing filter operations.</param>
+        /// <param name="alias">An alias for the class, if not provided the classes short name is used</param>
+        public static void WhitelistFilter(Type filterClassType, string alias = null)
```

Comment:
AddFilters is plural because a class can contain multiple filters. So why is this method singular? Or even simply AddFilters.Whitelist ?

Reply:
Changed to Whitelist.
I originally opted for `AddFilters.WhitelistFilter` over `Template.WhitelistFilter` as it is feels like better encapsulation and avoids adding more complexity in Template.